worker-document-archetype/documentation/WorkerArchetypeUsage.md

```diff
@@ -139,7 +139,23 @@ Generate the new Document Worker Aggregator from the `worker-document-archetype`
 The foundations for your new Document Worker is now set up. The generated project will contain the following submodules:
 
 - `<artifactId>` - submodule containing the Worker's backend code.
-- `<artifactId>-container` - submodule containing the Worker's container and testing configurations.
+- `<artifactId>-container` - submodule containing the Worker's container and [testing configurations](#document-worker-default-configuration).
+
+## Document Worker Default Configuration
+
+The `<artifactId>-container` submodule contains a default JavaScript configuration file.
+Configuration variables check an environment variable for a value. If no value is found for an environment variable a default value is set if applicable.
+
+### DocumentWorkerConfiguration
+
+The default Document Worker configuration file checks for values as below;
+
+| Property | Checked Environment Variables | Default               |
+|----------|-------------------------------|-----------------------|
+| outputQueue  |  `CAF_WORKER_OUTPUT_QUEUE` | Use environment variable `CAF_WORKER_BASE_QUEUE_NAME` + "-out". <br> Or use environment variable `CAF_WORKER_NAME` + "-out". <br> Else use "worker-out".  |
+| threads  |  `CAF_WORKER_THREADS` | 1  |
```

Comment:
Can you also include and talk about the `failureQueue` option that was added?  And add it to the JavaScript obviously - it should default to `undefined`.